src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp

```diff
@@ -464,8 +486,8 @@ void medAbstractDatabaseImporter::importData()
 
     if ( !d->indexWithoutImporting )
     {
-        QString subDirName = "/" + patientId;
-        QString imageFileNameBase =  subDirName + "/" +  seriesId;
+        QString subDirName = QDir::separator() + patientId;
```

Comment:
oh woa, are you ok to create a PR on medInria (3.2) for these `QDir::separator()`? It could be needed before 4.0

Reply:
Alright